ebb_lint/test/test_flake8.py

```diff
@@ -76,276 +76,277 @@ def scan_for_checkers(request):
 
 
 all_sources = [
-    '',
+    pytest.param(s) for s in [
```

Comment:
`all_sources` is a mix of params and conditions like `skipif`, so I couldn't apply `pytest.params(...)` to all of them at the end